modules/Bio/EnsEMBL/Variation/DBSQL/PhenotypeFeatureAdaptor.pm

```diff
@@ -716,6 +719,7 @@ sub fetch_all_by_phenotype_accession_source {
 
   throw('phenotype ontology accession argument expected') if(!defined($accession));
  
+  my $ontologyFlag = $self->_include_ontology();
```

Comment:
Sorry, it's missing the line `my $attribFlag =  $self->_include_attrib();` and `$self->_include_attrib($attribFlag);` line 734

Reply:
Well spotted, thanks! Found two more, fixed in next commit.